macros/ano-20macros.cfg

```diff
@@ -493,6 +498,27 @@
         {MSGm_Lorin _"Oh, for shame! He was more heavily wounded than I thought. He just passed away so suddenly... sorry..."}
     [/event]
 
+    [event]
+        {KILLEDBY (Robert the Hefty) (Reme Carrenemoe)}
+        {MSGm_Reme _"This is for your outlaws' attack on the Carrenemoe clan, and my capture!"}
+        {MSGa_Gawen _"Wait, Reme, I had some questions I wanted to ask him!"}
+        [kill]
+            id=Robert the Hefty
+            fire_event=no
+            animate=yes
+        [/kill]
+        {MSGm_Reme _"I apologize, Gawen, the heat of battle got to me... I'll try not to let it happen again."}
+    [/event]
+
+    [event]
+        {KILLEDBY (Robert the Hefty) (Rob Roe)}
+        #po: use of "me" instead of "my" here (and in Rob Roe's response) is to give him a bit of an accent:
+        {MESSAGE (Robert the Hefty) () (Roberty the Hefty) _"Wait a minute... Rob Roe? What are you doing, me fellow Rob? What happened to our bandit fellowship?"}
+        #po: "'snothing" is a contraction for "it's nothing":
+        {MESSAGE (Rob Roe) (portraits/humans/thug.png) (Rob Roe) _"Oh, you know how things go, me fellow Rob... allegiances change. 'snothing personal, you see... strictly business."}
+        {MESSAGE (Robert the Hefty) () (Roberty the Hefty) _"Figures, I guess... still... the irony..."}
```

Comment:
The other lines are okay, but I don't really like this one. Maybe you can just get rid of it or rewrite it.

Reply:
How about just "Oh well, it figures..."?